modular/manager/manage_task.go

```diff
@@ -332,6 +337,32 @@ func (m *ManageModular) HandleReplicatePieceTask(ctx context.Context, task task.
 }
 
 func (m *ManageModular) handleFailedReplicatePieceTask(ctx context.Context, handleTask task.ReplicatePieceTask) error {
+	if handleTask.GetNotAvailableSpIdx() != -1 {
```

Comment:
Is it possible that multi secondary SP failed, and we create multi retry tasks here?

Reply:
While the primary sp replicating to sps concurrently, if there is failure occurs, it would stop all replication immediately to save the cost. So it would only know the secondary sp which fail firstly if there are multiple sp unhealthy.  Then the task will be removed from replicate-queue and a new GVG excluding previous failed SP will be used for new replicate-task, added into the replicate queue to start replication. there wouldn't be multiple tasks for the same object replication in the same time.